src/main/java/uk/gov/ons/ctp/integration/contactcentresvc/endpoint/AddressEndpoint.java

```diff
@@ -43,8 +45,21 @@ public AddressEndpoint(final AddressService addressService) {
    */
   @RequestMapping(value = "", method = RequestMethod.GET)
   public AddressQueryResponseDTO getAddressesBySearchQuery(
-      @Valid AddressQueryRequestDTO addressQueryRequest) {
+      @Valid AddressQueryRequestDTO addressQueryRequest) throws CTPException {
     log.with("requestParams", addressQueryRequest).info("Entering GET getAddressesBySearchQuery");
+
+    String addressQueryInput =
+        addressQueryRequest.getInput().trim().replaceAll("'", "").replaceAll(",", "");
```

Comment:
Question : have we been told to strip out single quotes ?  Can't see that in the AIMS validation Scala.

Reply:
No - inner quotes are allowed, as long as the 5 rule is followed. So 'a(space)(space)(space)z' is allowed through and AI does not blow chunks